solvers/skglm.py

```diff
@@ -39,7 +51,7 @@ def run(self, n_iter):
         )
 
     def get_result(self):
-        return dict(w=self.w)
+        return dict(beta=self.w)
```

Comment:
define self.beta instead of self.w for harmonization (also true for other solvers); this will be less confusing